WeatherApp/WeatherApp/HomeScreen/HomeScreenView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+struct HomeScreenView: View {
+    @ObservedObject var homeScreenViewModel = HomeScreenViewModel()
```

Comment:
Get familiar with SwiftUI property wrappers - https://swiftuipropertywrappers.com

In this case where we are initialising an `ObservableObject` inside the View itself we would want to use `@StateObject`

`@ObservedObject` is generally used when we are passing a reference from already created `ObservableObject` to the view. Usually through its constructor.